src/pbkdf2.rs

```diff
@@ -133,12 +133,14 @@ use {constant_time, digest, hmac, polyfill};
 ///
 /// `derive` panics if `iterations < 1`.
 ///
-/// `derive` panics if `out.len()` is larger than the output length of the
-/// digest function used by the PRF algorithm.
+/// `derive` panics if `out.len()` is larger than 2^32 - 1, per the
```

Comment:
Above, the existing documentation says:

> out.len()` must be no larger than the output length of the digest function used in the PRF algorithm. This limit is more strict than what the		 used in the PRF algorithm. This limit is more strict than what the
specification requires. As noted at https://github.com/ctz/fastpbkdf2,		 specification requires.

That part of the documentation also needs to be corrected.